src/ocm/components/clusterConfiguration/networkConfiguration/NetworkConfiguration.tsx

```diff
@@ -210,12 +211,15 @@ const NetworkConfiguration = ({
     [cluster.cpuArchitecture, cluster.openshiftVersion, featureSupportLevelData, isDualStack],
   );
 
+  const { supportedPlatformIntegration } = useClusterSupportedPlatforms(cluster.id);
```

Comment:
Two questions:
- So we are not going to use FeatureSupportLevels to determine when to block it, right?
- As long as the hosts are from Nutanix, UMN cannot be used, even if no integration with the platform is selected? IF that is the case, maybe the tooltip could be more specific

Reply:
- So we are not going to use FeatureSupportLevels to determine when to block it, right?
True. BE won't change the featureSupportLevels to indicate this is supported or not. We need to block in the UI.

- As long as the hosts are from Nutanix, UMN cannot be used, even if no integration with the platform is selected? IF that is the case, maybe the tooltip could be more specific
@jkilzi WDYT?

Reply:
IMO the text provided by Allison is accurate. You cannot use UMN with Nutanix.